compiler/vm/vmdef.nim

```diff
@@ -614,7 +629,7 @@ template types*(c: TCtx): untyped =
   ## Transition helper
   c.typeInfoCache.types
 
-proc init(cache: var TypeInfoCache) =
+proc init*(cache: var TypeInfoCache) =
```

Comment:
maybe I'm missing something, but shouldn't we call the template instead of simply defining it in the body?

Reply:
nevermind... github cut off the rest of the definition... sigh